core/proto/keytransparency_v1_types/keytransparency_v1_types.proto

```diff
@@ -236,3 +236,19 @@ message GetMutationsResponse {
   // results.
   string next_page_token = 7;
 }
+
+// GetMonitoringRequest contains the input parameters of the GetMonitoring APIs.
+message GetMonitoringRequest {
+  // start specifies the start epoch number from which monitoring results will
+  // be returned (ranging from [start, latestObserved] and starting at 1).
+  int64 start = 1;
+}
+
+message GetMonitoringResponse {
+  // smr contains the map root for the sparse Merkle Tree signed with the
```

Comment:
Thanks! I've updated your comment to clarify that the "invalid mutations" case is only from epoch e to e+1. My understanding still is that the "storage-less" monitor doesn't have all information if it doesn't additionally store the inclusion proofs at each epoch e. As the monitor can not know in advance (at epoch e) if there will be a problem in e+1. Is that correct or do I miss sth?